test-data/helloservice/helm/templates/deployment.yaml

```diff
@@ -26,6 +27,6 @@ spec:
           httpGet:
             path: /
             port: 9000
-          initialDelaySeconds: 60
+          initialDelaySeconds: 10
```

Comment:
could we please set it to 30 seconds. the demo use case works best with this setting

Reply:
done